Getting_started es-translation

lh1008 opened the pull request:
Hey @erciccione 

Here's the es_getting_started translation. Pinging @BlackLotus64 to see if he can check this out. Will try to search for a reviewer, if you know about someone please let him know about this PR. 

Thanks.

BlackLotus64 commented:
Working from home currently, and yes, absolutely, will have a chance in my afternoon in 8 and half hours.

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+
+La comunidad descentralizada de Monero está formada por un grupo de individuos muy dinámico y grupos propagados alrededor del planeta. Un hecho poco conocido y emocionante es que Monero tiene la tercera cuenta de colaboradores más alta en todas las criptomonedas. Adicionalmente, hay muchos grupos de trabajo y proyectos que han evolucionado de la mano de Monero y es relativamente fácil empezar a a ...
+
+Por favor usen esto como un recurso para aprender sobre Monero, conocer sobre su ecosistema y como PUEDEN involucrarse. Bienvenidos a Monero. Esperamos que se queden.
```
_Por favor usen esto como un recurso para aprender sobre Monero, conocer sobre su ecosistema y **cómo** PUEDEN involucrarse._

I would change the "PUEDEN involucrarse" to "PUEDES involucrarte" but is not a grammatical mistake, so it's fine.

lh1008 replied:
Changed.

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+
+### _Desarrolladores_
+
+Si eres un desarrollador, te invitamos a que te unas a algún equipo.
```
_Si eres un desarrollador, te invitamos a que te unas **al equipo**._

Since it just says, "join the team", not "join a team" in the original file.

lh1008 replied:
Changed.

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+
+Si eres un desarrollador, te invitamos a que te unas a algún equipo.
+
+La principal manera de empezar, es familiarizarse con la base del código, el estándar del protocolo cryptonote e interactuar con otros desarrolladores. Todos los desarrolladores se comunican a través de IRC en #monero-dev o #monero en los servidores de freenode. Encuentra aquí recursos para que los desarrolladores puedan empezar:
```
It's a suggestion, I believe that _base del código_ it's more known as _código base_. Used Google Translator and it gave the first result only, but the meaning would be the same.

lh1008 replied:
Yeah, it also works. It's okay, I'll change it anyway. Changed.

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+La principal manera de empezar, es familiarizarse con la base del código, el estándar del protocolo cryptonote e interactuar con otros desarrolladores. Todos los desarrolladores se comunican a través de IRC en #monero-dev o #monero en los servidores de freenode. Encuentra aquí recursos para que los desarrolladores puedan empezar:
+
+- Código Fuente - [github.com/monero-project](https://github.com/monero-project)
+- Asuntos(Issues) abiertos - [github.com/monero-project/monero/issues](https://github.com/monero-project/monero/issues)
+- El ecosistema de Monero - [github.com/monero-ecosystem](https://github.com/monero-ecosystem)
```
**Asuntos (Issues)**, there should be a space, shouldn't be?

Another suggestion, **Ecosistema de Monero** would be better, to follow the other bullets translations: is not _**El** laboratorio de investigación de Monero_, and so on.

lh1008 replied:
You're totally right. Changed. Thanks

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+
+### _Equipos de Trabajo_
+
+Abajo encontrarás la lista de los equipo de trabajo de Monero que se actualizaran a medida que nuevos equipos se establezcan. Si tienes una idea de un proyecto para apoyar a Monero que no este en la lista, toma la iniciativa y empieza. Existen algunos recursos disponibles para la comunidad de Monero para ayudar al desarrollo efectivo de los equipos de trabajo y sus operaciones. Si tienen pregunta ...
```
_Abajo encontrarás la lista de los equipo de trabajo de Monero que se **actualizarán** a medida que nuevos equipos se establezcan. Si tienes una idea de un proyecto para apoyar a Monero que no **esté** en la lista, toma la iniciativa y empieza. _

I would rephrase the next to:

_Si tienes preguntas, ve a alguno de los foros y pregunta lo que necesites. Tenemos decenas de colaboradores que pueden ayudarte a empezar._

lh1008 replied:
Suggestions changed. :)

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+_¿Dónde empezar?_ [r/MoneroCommunity](https://www.reddit.com/r/MoneroCommunity/)
+
+#### Localización
+Traducción de materiales relacionados con Monero. Si hablan otro idioma, revisen esto.
```
I feel like the translation is using a way to speak to the reader in plural, in example here:

_Si hablan otro idioma, revisen esto._

The original file, as per language (English) uses more like a "you", singular way to refer to the reader, not in plural. It may be interpreted different but a text usually talks to you in singular rather than plural. The main point would be:

_Si hablas otro idioma, revisa esto._

It's open to review since you used that way in other parts, it can be just left as is.

lh1008 replied:
Yeah, I was trying not to translate in singular but while I was reading it again it reads weird when translated to plural instead of singular. I'm making the changes. Thanks.

BlackLotus64 replied:
It's common when translating, but you'll also need to be on the other side of the translation, thanks lh1008.

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+_¿Dónde empezar?_ [taiga.getmonero.org/project/sgp-monero-meetup-kit](https://taiga.getmonero.org/project/sgp-monero-meetup-kit/)
+
+#### Integraciones
+Colaborar para poder integrar a Monero en una variedad de servicios, esto incluye procesadores de pagos, casas de cambio descentralizadas y más.
```
Suggestion to improve and reduce phrase:

_Colaborar para poder integrar a Monero en una variedad de servicios, **incluyendo** procesadores de pagos, casas de cambio descentralizadas y más._

lh1008 replied:
Yep, changed.

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+Colaborar para poder integrar a Monero en una variedad de servicios, esto incluye procesadores de pagos, casas de cambio descentralizadas y más.
+_¿Dónde empezar?_ [taiga.getmonero.org/project/serhack-monero-integrations](https://taiga.getmonero.org/project/serhack-monero-integrations/) & #monero-integrations
+
+#### Labratorio de transmisiones de Monero
```
It is **Laboratorio**, not Labratorio, it's a typo error.

lh1008 replied:
Oups, thanks xD

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+Apuntamos a resolver los problemas relacionados a la transmisión de las transacciones de Monero a través una infraestructura descentralizada de código abierto (redes mesh, satélites, etc.)
+_¿Dónde empezar?_ [taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/108](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/108)
+
+### _Únete a la conversación e Involucrate_
```
_Únete a la conversación e **Involúcrate**_

lh1008 replied:
Fixed.

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+- Dona al CCS (Sistema de Financiación Colectiva de la Comunidad) - [ccs.getmonero.org](https://ccs.getmonero.org/)
+- Minar Monero - [Ir a la sección](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#workgroups)
+- Únete a algún equipo de trabajo - [Ir a la sección](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#workgroups)
+- Contribuir a futuras versiones del libro Mastering Monero (Dominando a Monero) - [github.com/monerobook/monerobook](https://github.com/monerobook/monerobook) y [masteringmonero.com](https://masteringmonero.com/)
+- Responder preguntas en Stack Exchange - [monero.stackexchange.com](https://monero.stackexchange.com/)
+- Conviertete en un Mercaderista de guerrilla - [monerooutreach.org/guerrilla-toolkit.html](https://www.monerooutreach.org/guerrilla-toolkit.html)
```
To follow verbs:

_Contribuye_.
_Responde_.

And this is just missing:

_Conviértete_

lh1008 replied:
Fixed.

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+Abajo encontrarás la lista de los equipo de trabajo de Monero que se actualizaran a medida que nuevos equipos se establezcan. Si tienes una idea de un proyecto para apoyar a Monero que no este en la lista, toma la iniciativa y empieza. Existen algunos recursos disponibles para la comunidad de Monero para ayudar al desarrollo efectivo de los equipos de trabajo y sus operaciones. Si tienen pregunta ...
+
+#### Desarroladores
+Repositorio de desarrollo de Monero, incluidos el daemon, la CLI, GUI y otros software cruciales.
```
Grammatical structure suggests, since _software_ is now a Spanish word (anglicism):

_Repositorio de desarrollo de Monero, incluidos el daemon, la CLI, GUI y otros *softwares** cruciales._

lh1008 replied:
Fixed, I was confused while trrying to add an 's' and the end of the softwares word. Thank you for clarifying this to me.

BlackLotus64 replied:
Yes, I know, it was a little weird, but, well, technical side of the language 😃

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+_¿Dónde empezar?_ [taiga.getmonero.org/project/michael-rfc-hwallet-1-implementation]](https://taiga.getmonero.org/project/michael-rfc-hwallet-1-implementation/)
+
+#### Soporte
+Ayudando a los usuarios por una variedad de razones, incuye la configuración de monederos, la conexión a nodos, restauración de cuentas, transferencia de fondos y más.
```
+Ayudando a los usuarios por una variedad de razones, **incluye** la configuración de monederos, la conexión a nodos, restauración de cuentas, transferencia de fondos y más.+

lh1008 replied:
Thanks.

BlackLotus64 commented on monero-outreach-docs/translations/es/archive/getting_started_es.md:
```diff
@@ -0,0 +1,197 @@
+_¿Dónde comenzar?_ [github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md](https://github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md)
+
+#### RIAT - Instituto para la Economía futura
+Promoviendo proyectos de cryptomonedas, incluyendo a Monero. Instituto basado en Vienna.
```
+Promoviendo proyectos de **criptomonedas**, incluyendo a Monero. Instituto basado en Vienna._

lh1008 replied:
You're right on this.

lh1008 commented:
Hey @BlackLotus64 

Added the changes and fixed the grammatical mistakes. Take a look at them and make the approval so we merge them :). Thanks man.

lh1008 commented:
Oh @BlackLotus64 just noticed about the commit of the _rpc_pay_es.md_. I will have to fix it once I push the other translation. Thanks for noticing it xP.
